doc/examples/streamline_registration.py

```diff
@@ -208,43 +213,40 @@
 """
 
 from dipy.io.streamline import load_trk
-
 streamlines, hdr = load_trk('lr-superiorfrontal.trk')
 
 
 """
 We then apply the obtained deformation field to the streamlines. We first
-apply the non-rigid warping and then apply the computed affine transformation
-whose extents must be corrected to account for the different voxel grids of
-the moving and static images:
+apply the non-rigid warping and simultaneously apply a computed rigid affine
+transformation whose extents must be corrected to account for the different
+voxel grids of the moving and static images:
 
 """
+from dipy.tracking.streamline import deform_streamlines
 
-from dipy.tracking.streamline import deform_streamlines, Streamlines
-
-from dipy.tracking.utils import move_streamlines
-
-warped_streamlines = \
-    deform_streamlines(streamlines,
-                       deform_field=mapping.get_forward_field(),
-                       stream_to_current_grid=moving_affine,
-                       current_grid_to_world=mapping.codomain_grid2world,
-                       stream_to_ref_grid=static_affine,
-                       ref_grid_to_world=mapping.domain_grid2world)
+# Create an isocentered affine
+target_isocenter = np.diag(np.array([-vox_size, vox_size, vox_size, 1]))
```

Comment:
Should we explain the negative sign in the first dimension ? It is may be evident, but I do not see the rationale @dPys .

Reply:
We can and may want to. The reason is that we need to create a dummy affine in LAS orientation of the given voxel resolution (2mm) that is the mirror image of that which was used to create the forward deformation field.

Reply:
Got it. Thanks for the explanation Derek.